Files/MultilingualResources/Files.hi-IN.xlf

```diff
@@ -981,61 +972,61 @@
         </trans-unit>
         <trans-unit id="SettingsOnStartupOpenAddPageButton.Content" translate="yes" xml:space="preserve">
           <source>Add a new page</source>
-          <target state="new">Add a new page</target>
+          <target state="translated">एक नया पन्ना जोड़ें</target>
```

Comment:
```suggestion
          <target state="translated">एक नया पृष्ठ जोड़ें</target>
```

Reply:
भाई मुझे नहीं लगता कोई पृष्ठ इस्तेमाल करता है पन्ना के जगा‍

Reply:
@soumyamahunt Yes, it's not used as much "while speaking", but in software we should use the "shudh hindi" words. "पृष्ठ" is a "तत्सम" word, i.e. from Sanskrit. It is used everywhere in written literature 😊.

Reply:
I think software is written so it can be used by common people, while written literature are used for academic purposes. I think we should write what conveys the functionality to the broader masses. Both पन्ना and पृष्ठ are correct, but पन्ना is used more hence we should keep that.

Reply:
@soumyamahunt I understand what you are saying but "पन्ना" is almost never used in written form. It's deemed unprofessional to write it instead of "पृष्ठ". And it's not like it's a completely arcane word that nobody understands. Everyone who has ever studied hindi knows this word, since it is the standard in writing. Using "पृष्ठ" gives a more professional vibe. This is my opinion, so you you get to make the final call.😊